cmd/bedSplit/bedSplit.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/vertgenlab/gonomics/bed"
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"log"
+	"os"
+	"strings"
+)
+
+type Settings struct {
+	Mode    string
+	InFile  string
+	OutDir  string
+	GzipOut bool
+}
+
+func bedSplit(s Settings) {
+	var err error
+	var foundInMap bool
+	if _, err = os.Stat(s.OutDir); os.IsNotExist(err) {
+		err = os.Mkdir(s.OutDir, 0755)
+		exception.PanicOnErr(err)
+	}
+
+	records := bed.GoReadToChan(s.InFile)
+	switch s.Mode {
+	case "byName":
+		var seenNames = make(map[string]*fileio.EasyWriter, 0)
+		for v := range records {
+			v.Name = strings.Replace(v.Name, "/", "_", -1)
```

Comment:
Is the replacement of "/" with _ in bed name fields because output file names with "/" would mess up the path?
I think it is possible to retain "/" in the bed files, but just change "/" to _ in file names, but it may not be a necessary revision!

Reply:
Yes this will disrupt paths. I went back and forth stylistically on whether to update "/" in the bed files, but I think this may be programmatically cleaner and more interpretable for the user. Hopefully a rare corner case as well!